pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -526,6 +526,9 @@ func (l *listUsersQuery) expandExclusion(
 		}
 	}
 
+	var errs error
+	errs = errors.Join(errs, errsBase)
+	errs = errors.Join(errs, errsSub)
```

Comment:
I think you can join multiple?

```suggestion
	errs = errors.Join(errs, errsBase, errsSub)
```